test/regression.uts

```diff
@@ -809,6 +809,11 @@ assert SNMP in z
 x = UDP()/SNMP()
 assert x.sport == x.dport == 161
 
+= Test snmpwalk()
+if WINDOWS or os.getenv("SCAPY_SUDO") == "sudo":
```

Comment:
Sorry I missed that but you should remove this test and add `~ netaccess` instead.

Reply:
Done